packages/slate-editor/src/modules/editor-v4-story-bookmark/lib/createStoryBookmark.ts

```diff
@@ -0,0 +1,29 @@
+import type { StoryBookmarkNode } from '@prezly/slate-types';
+import { STORY_BOOKMARK_NODE_TYPE, StoryBookmarkLayout } from '@prezly/slate-types';
+import { v4 as uuidV4 } from 'uuid';
+
+type RequiredProps = Pick<StoryBookmarkNode, 'story'>;
+type OptionalProps = Omit<StoryBookmarkNode, keyof RequiredProps>;
```

Comment:
I'd suggest explicitly omitting `type` and `children` properties to prevent human mistakes.